.travis.yml

```diff
@@ -103,8 +107,6 @@ script:
         done;
       fi
     - cd .. && mv julia julia2
-    - cp /tmp/julia/lib/julia/sys.ji local.ji && /tmp/julia/bin/julia -J local.ji -e 'true' &&
-        /tmp/julia/bin/julia-debug -J local.ji -e 'true' && rm local.ji
```

Comment:
Agreed. Even if slightly redundant I don't see much benefit in removing this test.